railties/lib/rails/generators/rails/app/templates/config/application.rb

```diff
@@ -32,5 +32,6 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    config.index_nested_attributes_errors = true
```

Comment:
This doesn't work. We don't have any global config with this name. We should set it inside the `active_record` namespace. Also it should not be inside the config/application and I don't even think it should be true by default. Let just document it on the configuring guide and not include in the generated application.